src/routes/(console)/project-[project]/overview/platforms/createWeb.svelte

```diff
@@ -0,0 +1,344 @@
+<script lang="ts">
+    import { Wizard } from '$lib/layout';
+    import { invalidate } from '$app/navigation';
+    import { createPlatform } from './wizard/store';
+    import { Dependencies } from '$lib/constants';
+    import {
+        Code,
+        Layout,
+        Icon,
+        Typography,
+        Fieldset,
+        InlineCode,
+        Card,
+        Button
+    } from '@appwrite.io/pink-svelte';
+    import { Form } from '$lib/elements/forms';
+    import {
+        IconVue,
+        IconAppwrite,
+        IconSvelte,
+        IconReact,
+        IconNuxt,
+        IconInfo,
+        IconExternalLink
+    } from '@appwrite.io/pink-icons-svelte';
+    import { page } from '$app/stores';
+    import { type ComponentType, onMount } from 'svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { addNotification } from '$lib/stores/notifications';
+    import { fade } from 'svelte/transition';
+    import ConnectionLine from './components/ConnectionLine.svelte';
+    import OnboardingPlatformCard from './components/OnboardingPlatformCard.svelte';
+    import { PlatformType } from '@appwrite.io/console';
+    import ReactFrameworkIcon from './components/ReactFrameworkIcon.svelte';
+    import SvelteFrameworkIcon from '$routes/(console)/project-[project]/overview/platforms/components/SvelteFrameworkIcon.svelte';
+    import NuxtFrameworkIcon from '$routes/(console)/project-[project]/overview/platforms/components/NuxtFrameworkIcon.svelte';
+    import NextjsFrameworkIcon from '$routes/(console)/project-[project]/overview/platforms/components/NextjsFrameworkIcon.svelte';
+    import VueFrameworkIcon from '$routes/(console)/project-[project]/overview/platforms/components/VueFrameworkIcon.svelte';
+    import NoFrameworkIcon from './components/NoFrameworkIcon.svelte';
+
+    export let key;
+
+    let showExitModal = false;
+    let isPlatformCreated = !!key;
+    let isCreatingPlatform = false;
+    let connectionSuccessful = false;
+    let isChangingFramework = false;
+
+    const projectId = $page.params.project;
+
+    const updateConfigCode = `APPWRITE_PROJECT_ID = "${projectId}"
+APPWRITE_PUBLIC_ENDPOINT = "${sdk.forProject.client.config.endpoint}"
+        `;
+    type FrameworkType = {
+        key: string;
+        label: string;
+        icon: ComponentType;
+        smallIcon: ComponentType;
+        portNumber: number;
+    };
+    export let platform: PlatformType = PlatformType.Flutterandroid;
+    export let selectedFrameworkKey: string | undefined = key ? key : undefined;
+
+    let frameworks: Array<FrameworkType> = [
+        {
+            key: 'svelte',
+            label: 'Svelte',
+            icon: SvelteFrameworkIcon,
+            smallIcon: IconSvelte,
+            portNumber: 5173
```

Comment:
unrelated but assuming most of these use `vite`, we could just hardcode a port there in the `vite.config.ts` so we don't have to manage these ourselves. This should be fine for now though.